docs/python/unit-testing.md

```diff
@@ -70,20 +70,20 @@ You can also configure pytest using a `pytest.ini` file as described on [PyTest
 | "python.unitTest.nosetestPath" | `"nosetests"` | Path to Nose. Use a full path if PyTest is located outside the current environment. |
 |  "python.unitTest.nodetestArgs" | `[]` | Arguments to pass to Nose, with each argument specified as an item in the array. See [Nose usage options](http://nose.readthedocs.io/en/latest/usage.html#options). |
 
-You can also configure Nose wuth a `.noserc` or `nose.cfg` file as described on [Nose configuration](http://nose.readthedocs.io/en/latest/usage.html#configuration).
+You can also configure nose wuth a `.noserc` or `nose.cfg` file as described on [Nose configuration](http://nose.readthedocs.io/en/latest/usage.html#configuration).
 
 
 ## Test discovery
 
-VS Code uses the currently enabled unit testing framework in to discover tests by applying the respective discovery patterns specified in the arguments option for that framework.
+VS Code uses the currently enabled unit testing framework to discover tests by applying the respective discovery patterns specified in the arguments option for that framework.
 
-For example, the default arguments for unittest include `-s . -p *_test.py`, meaning that unittest looks recursively, starting with the project folder (`-s .`), for all files with names matching the "*_test.py" pattern (`-p *_test.py`). You can specify a different starting folder after `-s`, and/or a different pattern after `-p`.
+For example, the default arguments for unittest include `-s . -p test*.py`, meaning that unittest looks recursively, starting with the project folder (`-s .`), for all files with names matching the "*_test.py" pattern (`-p *_test.py`). You can specify a different starting folder after `-s`, and/or a different pattern after `-p`.
```

Comment:
The pattern is actually "*test*.py" (not sure where I got the _). Can you then edit the other instances of _test here to be *test*?

Reply:
Did you mean `test.py` or `*test*`.py?

Reply:
it's `*test*.py` in the settings. GitHub comments italicized it for me without the backticks. :)